manifests/init.pp

```diff
@@ -34,6 +34,9 @@
 # @param service_bind_address
 #   Bind address for Keycloak service.
 #   Default is '0.0.0.0'.
+# @param management_bind_address
+#   Bind address for Keycloak management.
+#   Default is '0.0.0.0'.
```

Comment:
Puppet-strings will document the default values automatically when it generates REFERENCE.md, so this is redundant and will look silly in documentation (default value is documented twice).